nilearn/plotting/tests/test_img_plotting/test_img_plotting.py

```diff
@@ -38,14 +38,14 @@
 PLOTTING_FUNCS_3D = ALL_PLOTTING_FUNCS.difference(PLOTTING_FUNCS_4D)
 
 
-def _test_data_with_nans(img):
+def _test_data_with_nans(img, mni_affine=MNI_AFFINE):
```

Comment:
Since a fixture exists for this, can't you just do:

```suggestion
def _test_data_with_nans(img, mni_affine):
```
and remove `from nilearn.conftest import MNI_AFFINE`

This is also relevant for `demo_plot_roi` function in `test_plot_roi.py`

Reply:
let me check because I think that for this one "it is not that simple" (TM)
but "I may wrong" (story of my life)

Reply:
yup this function actually just adds nans to an image to then pass it to some other tests

so AFAICT it cannot use fixtures

Reply:
am renaming the function to match what it does